src/logic/operate.js

```diff
@@ -0,0 +1,33 @@
+import Big from 'big.js';
+
+const Operate = (numberOne, numberTwo, operator) => {
+  const bigNumberOne = Big(numberOne);
+  const bigNumberTwo = Big(numberTwo);
+  let result = {};
+  let total = 0;
+
+  switch (operator) {
+    case '÷':
+      total = bigNumberOne / bigNumberTwo;
+      result = { total, next: total, operator };
+      break;
+    case '-':
+      total = bigNumberOne - bigNumberTwo;
+      result = { total, next: total, operator };
+      break;
+    case '+':
+      total = bigNumberOne + bigNumberTwo;
+      result = { total, next: total, operator };
+      break;
+    case 'X':
+      total = bigNumberOne * bigNumberTwo;
+      result = { total, next: total, operator };
+      break;
+    case '%':
+      return { total: bigNumberOne, next: bigNumberTwo / 100, operator };
```

Comment:
Job well done but you don't have to do the calculating yourself, that where the use of `big` comes in -> [big js library](https://github.com/favio41/big-js#readme)
example: case division will be -> `total.minus(next)`